examples/modified_cuda_samples/vectorAdd/vectorAdd.cu

```diff
@@ -59,7 +59,7 @@ int main(void)
 
 	cuda::launch(
 		vectorAdd,
-		{ blocksPerGrid, threadsPerBlock },
+		{ cuda::grid_block_dimensions_t(blocksPerGrid), cuda::grid_block_dimensions_t(threadsPerBlock) },
```

Comment:
1. Why did you need this?
2. The two parameters have different types.

Reply:
If I don't have this then I get the following errors

> ./cuda-api-wrappers/examples/modified_cuda_samples/vectorAdd/vectorAdd.cu(60): error C23
> 97: conversion from 'int' to 'cuda::grid_dimensions_t' requires a narrowing conversion [...]
> ./cuda-api-wrappers/examples/modified_cuda_samples/vectorAdd/vectorAdd.cu(60): error C23
> 97: conversion from 'int' to 'cuda::grid_block_dimensions_t' requires a narrowing conversion [...]